build/charts/flow-aggregator/values.yaml

```diff
@@ -18,6 +18,9 @@ flowAggregatorAddress: ""
 recordContents:
   # -- Determine whether source and destination Pod labels will be included in the flow records.
   podLabels: false
+# -- HostAliases to be injected into the Pod's hosts file.
+# For example: [{"ip": "8.8.8.8", "hostname": "clickhouse.example.com"}]
```

Comment:
I was wrong before, it should be `[{"ip": "8.8.8.8", "hostnames": ["clickhouse.example.com"]}]` to match K8s

Can you also try to use backticks around the example so that the documentation renders better?

Reply:
Fixed. Thanks!